### ISSUE TITLE
What's the Point?

### ISSUE THREAD

**Issue by dolphin767** [iffn/iffns360ChairForVRChat#1]
If this only works for Desktop players, and not VR, then what's the point? This is needed for VR. Not a lot of people play on their Desktop. I found this on VRChat Prefabs is why I ask.

**Comment by iffn** [comment]
Many people play VRChat on desktop. It's is mostly a social game and people hang out using various platforms. Now VRC is even pushing smart phones as away of accessing it.

Many people get motion sickness in VR, especially when using vehicles. Me included. So while I like hanging out in VR, I prefer flying in desktop for example.

So this gives desktop players more freedom to look around. SaccFlight also uses this for example.

**Comment by dolphin767** [comment]
Fair enough, but I really need a prefab for this in VR. I wondered why it didn't work in VR. I asked for it to be updated to the latest SDK, and then when It didn't work I looked at the description again. I didn't see it listed as only Desktop before. Now I see it is. Thanks for the reply. I wasn't trying to be snarky or rude, just didn't understand why. Now I do. Is there any way you can make this work in VR?
